game_functions.h

```diff
@@ -20,6 +20,7 @@ double get_min_y(const game &g);
 // About players
 int count_alive_players(const game& g) noexcept;
 double get_nth_player_size(const game& g, const int i);
+color get_nth_player_color(const game& g, const int i);
```

Comment:
I would suggest to move this function above the `get_nth_player_size`, we plan to arrange all the free functions in lexical order, and by doing this you can reduce the amount of work of people who volunteer to sort the functions